### ISSUE TITLE
Bless Unleashed (1254120)

### ISSUE THREAD

**Issue by PyDario** [ValveSoftware/Proton#5173]
# Compatibility Report
- Name of the game with compatibility issues: Bless Unleashed
- Steam AppID of the game: 1254120

## System Information
- GPU: GTX 1070
- Driver/LLVM version: nvidia 470.63.01
- Kernel version: 5.14.5-200-tkg-bmq
- [System information](https://gist.github.com/PyDario/7a4bf749957e22f9894fcf9f93e94c45)
- Proton version: 6.3-6

## I confirm:
- [x] that I haven't found an existing compatibility report for this game.
- [x] that I have checked whether there are updates for my system available.

[protonlog](https://gist.github.com/PyDario/bb3a8b1c1fbf758bc5f36b1e063b51f4)

## Symptoms 
You will be kicked out of the game after 2 minutes. Most likely due to EAC.

## Reproduction
1. Start up the game and create a character
2. Join a world to when the cutscene of the game plays
3. After 2 minutes (cutscene included) you will be kicked out of the game

**Comment by PyDario** [comment]
# Additional pictures
EAC loading upon starting the game
![image](https://user-images.githubusercontent.com/56290011/134223863-a7e02b46-8dcd-439e-9775-9db51f7896bb.png)

Messages upon kick
First one was "Authentication timed out" followed shortly by this
![image](https://user-images.githubusercontent.com/56290011/134224358-f7b2b36b-ac6d-4907-af4d-7c15cdef4fc6.png)
Text says: "The connection to the server has been interrupted. Bless Unleashed is momentarily unvailable. Please try again later. If the problem persists, contact us at..."

**Comment by alyssiaf** [comment]
Hi Guys, 
Same problem occur for me! I migrated from PC (worked perfectly) to Linux(Proton)  Game run great but time out. 

Please correct the problem!!!!

**Comment by GloriousEggroll** [comment]
I've just been able to login and play the game for about 15 minutes without being disconnected \o/ 

Anyone else able to check this?
![blessunleashed](https://user-images.githubusercontent.com/11287837/136741142-90820226-e8c2-4a0f-8e79-b486a50ec8e8.png)

EAC is fully loaded:

```
176972.254:0320:0324:trace:loaddll:build_module Loaded L"Z:\\home\\tcrider\\.local\\share\\Steam\\steamapps\\common\\Bless Unleashed\\EasyAntiCheat\\EasyAntiCheat_x86.dll" at 10000000: native
```

**Comment by PyDario** [comment]
> Replying to https://github.com/ValveSoftware/Proton/issues/5173#issuecomment-939713224

I'm gonna check on this once I get back home but that'd be amazing if it worked.
Did you use proton-ge?

**Comment by GloriousEggroll** [comment]
yes, used proton-ge. just to note for eac testing my recent proton-ge builds have this commit removed from the build:

https://github.com/ValveSoftware/wine/commit/e672f0ce6039542e88f6e93230a63c37d6ffd227#diff-db2058d0bab60fbdc3c376a5dbdbfe93d27e0ccae7e120178d42c9ed1f1d134f

**Comment by PyDario** [comment]
> Replying to [#5173 (comment)](https://github.com/ValveSoftware/Proton/issues/5173#issuecomment-939713224)

Unfortunately, I was not able to confirm this. I've tried with both Proton 6.3-7 and Proton-GE 6.19 and it didn't work either way.

The anti-cheat dll was fully loaded too, if that's what it means
```
4786.999:02ac:02b0:trace:loaddll:build_module Loaded L"Z:\\home\\dario\\GAmes\\Linux\\steamapps\\common\\Bless Unleashed\\EasyAntiCheat\\EasyAntiCheat_x86.dll" at 10000000: native
```

However, the exe was not
```
4786.736:02a4:02a8:trace:loaddll:build_module Loaded L"Z:\\home\\dario\\GAmes\\Linux\\steamapps\\common\\Bless Unleashed\\EasyAntiCheat\\EasyAntiCheat_Setup.exe" at 00400000: native
```

**Comment by GloriousEggroll** [comment]
> Replying to https://github.com/ValveSoftware/Proton/issues/5173#issuecomment-940183406

:( it must have been the usual temporary EAC 'working' issue. I just received a timeout when I logged in a minute ago.
